LongWarOfTheChosen/LongWarOfTheChosen.x2proj

```diff
@@ -1,1112 +1,1116 @@
-    <Content Include="Src\LW_Overhaul\Classes\X2LWActivityCreation_Reinforce.uc" />
-    <Content Include="Src\LW_Overhaul\Classes\X2LWActivityCreation_SafestRegion.uc" />
-    <Content Include="Src\LW_Overhaul\Classes\X2LWActivityDetectionCalc.uc" />
-    <Content Include="Src\LW_Overhaul\Classes\X2LWActivityDetectionCalc_ProtectResearch.uc" />
-    <Content Include="Src\LW_Overhaul\Classes\X2LWActivityDetectionCalc_Rendezvous.uc" />
-    <Content Include="Src\LW_Overhaul\Classes\X2LWActivityDetectionCalc_Terror.uc" />
-    <Content Include="Src\LW_Overhaul\Classes\X2LWActivityDetectionCalc_TroopManeuvers.uc" />
-    <Content Include="Src\LW_Overhaul\Classes\X2LWAlienActivityTemplate.uc" />
-    <Content Include="Src\LW_Overhaul\Classes\X2LWTemplateModTemplate.uc" />
-    <Content Include="Src\LW_Overhaul\Classes\X2MissionNarrative_LWNarrativeSet.uc" />
-    <Content Include="Src\LW_Overhaul\Classes\X2MissionSet_LW.uc" />
-    <Content Include="Src\LW_Overhaul\Classes\X2MultiWeaponTemplate.uc" />
-    <Content Include="Src\LW_Overhaul\Classes\X2StrategyElement_AcademyUnlocks_LW.uc" />
-    <Content Include="Src\LW_Overhaul\Classes\X2StrategyElement_ContinentBonuses_LW.uc" />
-    <Content Include="Src\LW_Overhaul\Classes\X2StrategyElement_DarkEvents_LW.uc" />
-    <Content Include="Src\LW_Overhaul\Classes\X2StrategyElement_DefaultAlienActivities.uc" />
-    <Content Include="Src\LW_Overhaul\Classes\X2StrategyElement_DefaultRewards_LW.uc" />
-    <Content Include="Src\LW_Overhaul\Classes\X2StrategyElement_DefaultStaffSlots_LW.uc" />
-    <Content Include="Src\LW_Overhaul\Classes\X2StrategyElement_LWAlienAI.uc" />
-    <Content Include="Src\LW_Overhaul\Classes\X2StrategyElement_LWMissionSources.uc" />
-    <Content Include="Src\LW_Overhaul\Classes\X2StrategyElement_LWObjectives.uc" />
-    <Content Include="Src\LW_Overhaul\Classes\X2StrategyElement_MissionSiteDescriptions_LW.uc" />
-    <Content Include="Src\LW_Overhaul\Classes\X2StrategyElement_NewTechs_LW.uc" />
-    <Content Include="Src\LW_Overhaul\Classes\X2StrategyElement_PointsofInterest_LW.uc" />
-    <Content Include="Src\LW_Overhaul\Classes\X2TargetingMethod_LWBlasterLauncher.uc" />
-    <Content Include="Src\LW_Overhaul\Classes\X2TargetingMethod_LWEvacZone.uc" />
-    <Content Include="Src\LW_Overhaul\Classes\X2TargetingMethod_LWRocketLauncher.uc" />
-    <Content Include="Src\LW_Overhaul\Classes\XComGameState_Effect_Bombard.uc" />
-    <Content Include="Src\LW_Overhaul\Classes\XComGameState_Effect_Failsafe.uc" />
-    <Content Include="Src\LW_Overhaul\Classes\XComGameState_Effect_ImmediateMultiTarget.uc" />
-    <Content Include="Src\LW_Overhaul\Classes\XComGameState_Effect_JavelinRockets.uc" />
-    <Content Include="Src\LW_Overhaul\Classes\XComGameState_Effect_TransferMecToOutpost.uc" />
-    <Content Include="Src\LW_Overhaul\Classes\XComGameState_LWAlienActivity.uc" />
-    <Content Include="Src\LW_Overhaul\Classes\XComGameState_LWAl
```

Comment:
Hmmm...I think this may be a line ending issue. Can you restore the original line endings so that it doesn't look like the whole file has changed?

Reply:
I never touched that file so I do not know what happened here. What is it exactly I need to change back?

Reply:
The build automatically verifies that all the files under `LongWarOfTheChosen` are listed in the ModBuddy project file. Since you added some new files, the project file was updated.

What are you using to edit the files? Most text editors give an indication of what lime endings are in use and allow you to swap between them. Standard ones are DOS, Unix and Mac.

Reply:
Oh I was using sublime 3 editor.

Reply:
Ok so now this file seems to have proper line endings. The diffs only show the lines that were added instead of the whole file.